rcgen/src/lib.rs

```diff
@@ -1692,13 +1694,13 @@ impl Certificate {
 			.derive(&self.subject_public_key_info)
 	}
 	/// Get the certificate in DER encoded format.
-	pub fn der(&self) -> &[u8] {
+	pub fn der(&self) -> &CertificateDer<'static> {
```

Comment:
Could you also mention in the docs how this can be converted into a vec or byte slice so that it can be saved into a file, ideally with an example that we test or rustdoc links so that we notice the breakage.